pkg/config/constants.go

```diff
@@ -70,6 +70,10 @@ const (
 
 	// PVCCleanupPodMemoryLimit is the memory limit used for PVC clean up pods
 	PVCCleanupPodMemoryLimit = "32Mi"
+
+	// RoutingClassAnnotationInfix is the infix of the annotations of DevWorkspace that are passed down as annotation to the WorkspaceRouting objects.
+	// The full annotation name is supposed to be "<routingClass>.routingclass.controller.devfile.io/<anything>"
+	RoutingClassAnnotationInfix = ".routingclass.controller.devfile.io/"
```

Comment:
this annotation is propagated to routing (technically to workspacerouting) but not to routingclass.
Ingress implementation use the following format:
```
nginx.ingress.kubernetes.io/...
traefik.ingress.kubernetes.io/...
```
So, I think we can simplify annotation name without harm, like
`.routing.controller.devfile.io/`
or longer `.workspacerouting.controller.devfile.io/` (the CRD identifier is `workspacerouting.controller.devfile.io`). I prefer the first.